SampleGallery/SampleGallery.csproj

```diff
@@ -79,7 +79,7 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x64'">
     <DebugSymbols>true</DebugSymbols>
     <OutputPath>bin\x64\Debug\</OutputPath>
-    <DefineConstants>$(DefineConstants);DEBUG;TRACE;NETFX_CORE;WINDOWS_UWP</DefineConstants>
+    <DefineConstants>TRACE;DEBUG;SDKVERSION_10586;SDKVERSION_14393;SDKVERSION_15063;SDKVERSION_16299;SDKVERSION_17134;NETFX_CORE;WINDOWS_UWP;CODE_ANALYSIS</DefineConstants>
```

Comment:
I believe Visual Studio automatically put these here when you added the new #define via the UI.  We should delete these, and instead add a new line in SamplesCommon\SamplesCommon\CommonBuild.props with the new SDK version to conditionally define it as appropriate

Reply:
fixed in most recent PR